### PR TITLE
Terraform readme + variables.tf updated

### PR DESCRIPTION
**Pull request by S1mpySloth**
This pull request does 2 things:

Writes an initial terraform README - structured to accomodate future changes whilst documenting current available resources. 

Updates variables.tf with the new ECR repo URI's (This will of course also need to be included within your .tfvars

### CODE REVIEW

**Review by Fahi28 — APPROVED** [review]
looks good

File: terraform/README.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+C14_VPC       = "your_vpc_id"
+C14_SUBNET_1  = "your_subnet_id"
+ETL_ECR_URI   = "your_ecr_repo_uri"
+DATA_UPLOAD_ECR_URI = "your_ecr_repo_uri"
+DASHBOARD_ECR_URI = "your_ecr_repo_uri"
+```
+
+### Initialize Terraform
+
+To initialize the Terraform environment, run:
```
Comment (jiuliangut) [inline terraform/README.md:81]: uk spelling for consistency? (sorry to be annoying and pick up on this again!)

File: terraform/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+### Deploy the infrastructure
+
+Apply the Terraform configuration to deploy the infrastructure:
+
+```bash
+terraform apply
+```
+
+### Verify deployment
```
Comment (jiuliangut) [inline terraform/README.md:103]: Nice idea to have this section - makes the end goal clear :)

**Review by jiuliangut — COMMENTED** [review]
looks really good! just a quick fix on us/uk spelling otherwise good to go :)

File: terraform/README.md (hunk 1 above)
Comment (S1mpySloth) [reply terraform/README.md:81]: Im saddned as a Brit, I have done it againnn. Fixed!

**Review by Fahi28 — APPROVED** [review]
nice stuff

**Pull request merged**